### PR TITLE
Added a test that demonstrates the 500 error with a wrong 'Accept' header

### PR DESCRIPTION
**Pull request by ysavary**
Just added a test that demonstrates the 500 error when an wrong 'Accept' header is passed

### CODE REVIEW

File: apistar/hooks.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def render_response(handler: Handler,
         renderer = negotiate_renderer(accept, renderers)
         if renderer is None:
             raise exceptions.NotAcceptable()
+            # return http.Response(status=406)
```
Comment (drpoggi) [inline apistar/hooks.py]: Looks like this slipped in by mistake :)

Comment (ysavary) [reply apistar/hooks.py]: This commented line will make the test pass but it's certainly not the right way to fix the issue.

**Comment by drpoggi** [comment]
I think you may want to take a second look at this, `test_text_with_wrong_accept_header` is failing in the Travis build.

**Comment by ysavary** [comment]
@drpoggi i'm sorry, i tried but i don't know how to fix properly this issue. I added this failing test "test_text_with_wrong_accept_header" to demonstrate the issue.

**Comment by drpoggi** [comment]
Ah I see. Is there an existing Github issue tracking the error you're getting?

**Comment by ysavary** [comment]
It seems that NotAcceptable exception is not handled properly in this rendered context. A 2nd uncaught exception is propagated up to the root.

Returning a 406 response from here works but, in this case, i don't know how to call the error_handler to render the correct error response.

**Comment by tomchristie** [comment]
Closing this off given that 0.6 is moving to a framework-agnostic suite of API tools, and will no longer include the server. See https://discuss.apistar.org/t/api-star-as-a-framework-independant-tool/614 and #624.

**Pull request closed without merging**